omod/src/main/webapp/resources/htmlforms/071-IntegratedAntenatalRegister.xml

```diff
@@ -1055,6 +1063,14 @@
                         </p>
                     </td>
                 </tr>
+				<tr>
+					<td>
+						<label>Next appointment date</label>
```

Comment:
@paleu256 Add a validation rule, such that when the Next Appointment date is entered it has to be after the visit date. The ART encounter page I think has such a validation that you can copy and integrate here